.binder/postBuild

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+# binder post build script
+set -ex
+
+(cd docs && make html)
```

Comment:
```suggestion
cd docs && make html
```

Reply:
I did this so I don't have to do cd .. after